topics/sequence-analysis/tutorials/annotation-with-prokka/data-manager.yaml

```diff
@@ -0,0 +1,21 @@
+# configuration for fetch and index genomes
```

Comment:
If you do not need a data manager, you can then delete this file

Reply:
Thanks, deleted.